pkg/controller/operators/olm/operator.go

```diff
@@ -551,6 +551,12 @@ func (a *Operator) syncClusterServiceVersion(obj interface{}) (syncError error)
 	})
 	logger.Debug("syncing CSV")
 
+	if clusterServiceVersion.IsCopied() {
+		logger.Debug("skipping copied csv transition, schedule for gc check")
+		a.gcQueueIndexer.Enqueue(clusterServiceVersion)
```

Comment:
moved this here from `transitionCSV`  (b/c we can skip everything if it's copied)